src/main/perl/BlockdevFactory.pm

```diff
@@ -62,7 +63,12 @@ sub build
     elsif ($dev eq "tmpfs") {
         return NCM::Tmpfs->new(BASEPATH . $dev, $config);
     }
-    # WTF?
+    elsif ($dev =~ m!^vxvm/.*!) {
```

Comment:
the `.*` only makes the regex slower (but i see it's used before :see_no_evil: )